packages/lsp-extension/src/renderer.tsx

```diff
@@ -203,6 +204,7 @@ function BuildSettingForm(props: ISettingFormProps): JSX.Element {
     any,
     [hash: string, property: ISettingProperty]
   >(setProperty);
+  const textInputId = DOMUtils.createDomID() + '-line-number-input';
```

Comment:
Can you please wrap `textInputId` generation into `useRef` to ensure it is not regenerated on each render?

Reply:
Changes made.